### PR TITLE
Programmes

### PR DESCRIPTION
**Pull request by jamesjefferies**

### CODE REVIEW

File: app/controllers/programme_types_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class ProgrammeTypesController < ApplicationController
```
Comment (jamesjefferies) [inline app/controllers/programme_types_controller.rb]: Actually, I think this controller is now longer required and can be deleted

File: lib/tasks/do_everything.rake [hunk 2]
```diff
@@ -0,0 +1,11 @@
+namespace :do do
```
Comment (jamesjefferies) [inline lib/tasks/do_everything.rake]: Whoops, wondered where this has gone - popped this in to test new analytics releases, obvs. can be expanded on and in the end, pass around the aggregated meter collection...

**Comment by jamesjefferies** [comment]
Ah yes, meant to get rid of that instance variable in the view! Pre-holiday trying to get stuff done...

Happy for you to fix any nonsense which I committed, else to wait until my return! Could merge PR and I’ll refactor later...

**Comment by jamesalmond** [comment]
@jamesjefferies I've already worked through most of them. I was mainly submitting the review for my own notes!

**Pull request merged**